qa/api/src/test/java/com/epam/cm/tests/LoginTests.java

```diff
@@ -2,29 +2,86 @@
 
 import com.epam.cm.base.SimpleBaseTest;
 import io.restassured.http.ContentType;
+import io.restassured.response.Response;
 import org.junit.Test;
 
 import static io.restassured.RestAssured.given;
 
 public class LoginTests extends SimpleBaseTest {
 
+
     @Test
     public void positiveLoginTest(){
+        Response response1 =
+        given()
+                .contentType(ContentType.JSON)
+                .baseUri(config.baseHost)
+                .auth().preemptive().basic(config.speakerUser, config.speakerPassword)
+                .cookie("XSRF-TOKEN", response.cookie("XSRF-TOKEN"))
+                .header("X-XSRF-TOKEN", response.cookie("XSRF-TOKEN"))
+                .
+                        when()
+                .post( "/api/login")
+                .
+                        then().log().all()
+                .statusCode(200).extract().response();
+
+    }
+    @Test
+    public void negativeLoginTestInvalidLogin(){
```

Comment:
Please apply Data Driver for negative test. 
<dependency>
       	<groupId>com.tngtech.java</groupId>
       	<artifactId>junit-dataprovider</artifactId>
       	<version>1.5.0</version>
       	<scope>test</scope>
</dependency>

 https://dzone.com/articles/alternative-junit